modules/vba_documents/app/models/vba_documents/upload_submission.rb

```diff
@@ -10,6 +10,9 @@ class UploadSubmission < ApplicationRecord
     include SetGuid
     include SentryLogging
     include Webhooks
+
+    attribute :s3_deleted, default: false
```

Comment:
Note: Unlike a data migration, this won't default `s3_deleted` to `false` for previously created records. Those will still be `s3_deleted: nil`.

Reply:
yep, two options here, have the upcoming migration change any nils to false, or, as I was leaning towards, just manually change them myself after this PR is deployed.

Having the migration do the data changes is already hitting some issue, they have some kind of migration performance bot in place that encourages a manual change.

Reply:
Sorry, I probably missed your comment!